src/seedu/addressbook/AddressBook.java

```diff
@@ -210,12 +210,7 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
-        }
+        loopProgram();
```

Comment:
the name `loopProgram` may be a little vague (or misunderstanding). perhaps `runMainLoop`?